docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -147,38 +348,127 @@ support for the v1beta1 version of the Cluster API contract will be removed tent
-   - Considering the efforts to improve contract documentation in the last few releases, it is also highly recommended
-     to check the implementation of all the other rules.  
-2. Change the CRD annotation that document which Cluster API contract is implemented by your Provider.
+As a quick summary, transition should go through following phases:
 
-#### How to start using metav1.conditions
+#### Stage 1 
 
-We highly recommend providers define their future strategy for condition management as soon as possible.
+Add `status.v1beta2.conditions` to your API (existing conditions will remain at `status.conditions`)
 
-If providers choose to migrate `metav1.Conditions`, the process described in [Improving status in CAPI resources](https://github.com/kubernetes-sigs/cluster-api/blob/4a2e16a3d71d99a923b1f7f3a3efe03c299665fb/docs/proposals/20240916-improve-status-in-CAPI-resources.md)
-can be used as a reference about about how to implement a phased transition.
+If you are at this stage, you must use following util packages from CAPI v1.11 (or following releases, see note below):
+
+```go
+import (
+	...
+    conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"
+    v1beta2conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions/v1beta2"
+    patch "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch"
+)
+```
+
+- the `conditions` package alias provides access to utils for managing `clusterv1beta1.Conditions` in `status.conditions`
+- the `v1beta2conditions` package alias provides access to utils for managing `metav1.Conditions` in `status.v1beta2.conditions`
+- the `patch` package alias provides access to utils for patching objects in this phase.
+
+Important!
+All packages from the import above (all packages below `sigs.k8s.io/cluster-api/util/deprecated/v1beta1`) are going
+to be removed from CAPI when the v1beta1 removal will happen (tentative Aug 2026).
+
+#### Stage 2
+
+Create a new API version, swap old conditions and new conditions, implement conversions
+- Move old conditions from `status.conditions` to `status.deprecated.v1beta1.conditions`
+- Move new conditions from `status.v1beta2.conditions` to `status.conditions`
+
+If you are at this stage, you must use following util packages from CAPI v1.11 (or following releases, see note below):
+
+```go
+import (
+	...
+    "sigs.k8s.io/cluster-api/util/conditions"
+    v1beta1conditions "sigs.k8s.io/cluster-api/util/conditions/deprecated/v1beta1"
+    "sigs.k8s.io/cluster-api/util/patch"
+)
+```
+
+- the `conditions` package provides access to utils for managing `metav1.Conditions` in `status.conditions`
+- the `v1beta1conditions` package alias provides access to utils for managing `clusterv1beta2.Conditions` in `status.deprecated.v1beta1.conditions`
+- the `patch` package provides access to utils for patching objects in this phase
+
+Important!
+All package below `sigs.k8s.io/cluster-api/util/deprecated/` are going to be removed from CAPI when the
```

Comment:
I think you might want to update this one a bit. Currently it's stating all packages below util/deprecated will be removed.
1. if we aim for this path let's use deprecated/v1beta1
2. the import used in this example is not below util/deprecated

Reply:
Fixed, PTAL